prime-angular-frontend/src/app/modules/site-registration/pages/care-setting-page/care-setting-page.component.ts

```diff
@@ -170,15 +173,41 @@ export class CareSettingPageComponent extends AbstractCommunitySiteRegistrationP
   }
 
   protected submissionRequest(): NoContent {
-    if (this.formState.json.careSettingCode !== CareSettingEnum.DEVICE_PROVIDER) {
-      this.siteFormStateService.deviceProviderFormState.clearIndividualDeviceProviders();
-    }
-    if (this.formState.json.careSettingCode === CareSettingEnum.PRIVATE_COMMUNITY_HEALTH_PRACTICE) {
-      this.siteFormStateService.businessLicenceFormState.resetSiteId();
+
+    const sitePayload = this.siteFormStateService.json;
+    const request$ = this.siteResource.updateSite(sitePayload);
+
+
+    if (this.formState.json.careSettingCode === CareSettingEnum.PRIVATE_COMMUNITY_HEALTH_PRACTICE
+      && this.vendorConfig.find(v => v.code === this.formState.vendorCode.value).name === 'CareConnect'
+    ) {
+      const data: DialogOptions = {
+        title: 'Save & Continue',
```

Comment:
I think use of "&" is frowned on in PRIME labels and text ... change to `'Save and Continue confirmation'`

Reply:
Updated as suggested.